### DISCUSSION TITLE
👩🏻‍💻 Sunbird-RC Webinar Series : Session #1 (26th September) Accelerate Digital Transformation with the Power of Data Registries and Verifiable Credentials

### DISCUSSION THREAD

**Discussion by parthlawate** [Sunbird-RC/community#269] [Announcements]
Explore the power of data registries and verifiable credentials and
1) Transform the operational data needs of your organisation 
2) Empower your consumers to control their data
3) Enable seamless consent-based sharing for specific transactions

Join [Pramod Varma](https://www.linkedin.com/in/ACoAAAAsCc8Bui8fjIkq6jsqolsGWlPYb8rU5gs), CTO EkStep Foundation, Cofounder [Becknprotocol.io](http://becknprotocol.io/), Chief Architect Aadhaar & India Stack on the 26th of September at 6pm IST to know more.

Additionally in our “In Conversation” segment, Pramod Varma chats with [Daniel Goldscheider](https://www.linkedin.com/in/ACoAAAABNi4BGMrmDae_JWkitr-dyqBUVqxa6lY); Co-Founder & CEO, [Yes.com](http://yes.com/) and Lead Advocate of the "Open Wallet Initiative" on the vision behind an open-source engine to power interoperable wallets.

Register Now: https://lnkd.in/gUt2cDH8

**Comment by pramodkvarma** [comment]
For those who have missed, here is the recording of the event

https://www.youtube.com/watch?v=r86n8RPemaM

Thanks.
